src/kOS/UserIO/TelnetMainServer.cs

```diff
@@ -143,18 +170,26 @@ private void SetPermanentRealIPPermission(bool newValue)
         
         public void SetConfigEnable(bool newVal)
         {
+            if (bindAddr == null) // when nothing has been set yet, this will be the case.
```

Comment:
Don't make this change yourself.  I just had a thought about another way to implement it that would be a little cleaner.  I can do it as part of my review.